static/app/views/performance/transactionSummary/transactionOverview/eapChartsWidget.tsx

```diff
@@ -71,13 +74,19 @@ type EAPChartsWidgetProps = {
 };
 
 export function EAPChartsWidget({transactionName}: EAPChartsWidgetProps) {
-  const [selectedWidget, setSelectedWidget] = useState<EAPWidgetType>(
-    EAPWidgetType.DURATION_BREAKDOWN
-  );
   const location = useLocation();
+  const navigate = useNavigate();
+
   const spanCategoryUrlParam = decodeScalar(
     location.query?.[SpanIndexedField.SPAN_CATEGORY]
   );
+  const selectedChartUrlParam = decodeScalar(
+    location.query?.[SELECTED_CHART_QUERY_PARAM]
+  );
```

Comment:
Do you mind switching this to `useLocationQuery` instead? It's a tidier way of getting parameters off the URL, especially if there's more than one